content/docs/organizers/creating-emails.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Creating Email Campaigns
+description: Design and build email campaigns
+---
+
+# Creating Email Campaigns
+
+Create email campaigns to send to your contacts.
```

Comment:
maybe helps to clarify distinction between a campaign and the full contact list?

```suggestion
Create email campaigns to a set of contacts.
```